var/spack/repos/builtin/packages/root/package.py

```diff
@@ -55,6 +55,7 @@ class Root(CMakePackage):
     depends_on('cmake@3.4.3:', type='build')
     depends_on('pkg-config',   type='build')
 
+    depends_on('binutils')
```

Comment:
this does not seem to be related to the PR.

Reply:
That should have already made it into the repo. Weird